docs/rules/filename-case.md

```diff
@@ -57,3 +61,17 @@ Or set the `cases` option to allow multiple cases:
 	}
 ]
 ```
+
+### ignore
+
+An array of files to ignore. It’s set to [] by default. If provided, it must be an Array.
+
+```js
+"unicorn/filename-case": [
+	"error",
+	{
+		"case": "kebabCase",
+		"ignore": ["CHANGELOG.md", 'README.md']
```

Comment:
should use js file for example, maybe just `foo.js` `bar.js`

Reply:
let's do it